### PR TITLE
DOCS-1758: [TSLA-4906] Managed cluster deletion docs

### PR DESCRIPTION
**Pull request by jaderhs**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud/get-started/connect/cc-arch-diagram.mdx [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following diagram shows the major components in a managed cluster, followed
...
 | Fluentd                        | Open-source data collector for unified logging. Collects and forwards {{prodname}} logs (flows, DNS, L7) to log storage. | &bull; TCP 8080 to Guardian<br />&bull;  TCP 9080 from Prometheus API service |
 | Guardian                       | An agent running in each managed cluster that proxies communication between the {{prodname}} tunnel server and your managed cluster. Secured using TLS tunnels.<br /> | &bull; Port 9000 to tunnel server<br />&bull; TCP 6443 to Kubernetes API server<br />&bull; TCP 6443 from {{prodname}} components |
-| Installation endpoints         | Endpoints at  `*.calicocloud.io` and `*.`projectcalico.org`. | TCP 443 for both                                             |
+| Installation endpoints         | Endpoints at  `*.calicocloud.io` and `*.projectcalico.org`. | TCP 443 for both                                             |
```
Comment (ctauchen) [inline calico-cloud/get-started/connect/cc-arch-diagram.mdx:39]: good spot -- thanks!

File: calico-cloud/operations/cluster-management.mdx [hunk 2]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
 
 To connect new clusters to {{prodname}}, select Managed Clusters in the left nav, and click **Connect Cluster**.
 
-## Recover a deleted managed cluster
+## Delete a managed cluster
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx]: I think this title is misleading. We're not deleting a cluster so much as removing it from the list. 

I suggest we go with something like

`## Remove a cluster from the managed clusters list`

File: calico-cloud/operations/cluster-management.mdx [hunk 3]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
 
-If you inadvertenty delete a managed cluster, rerun the `curl` command and the cluster will reconnect to {{prodname}}.
+### Prerequisites
+
+- Only users with either the `Owner` or `Admin` [roles](../users/user-management.mdx) are permitted to delete managed clusters.
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx]: * You are signed in to Manager UI and you have [Owner or Admin privileges](../users/user-management.mdx).

File: calico-cloud/operations/cluster-management.mdx [hunk 4]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
+- The managed cluster must not be connected. For help with disconnecting your managed cluster, see [how to uninstall {{prodname}}](disconnect.mdx).
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx]: * The cluster you want to remove is [disconnected from Calico Cloud](disconnect.mdx).

File: calico-cloud/operations/cluster-management.mdx [hunk 5]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
+
+### Delete a managed cluster
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx]:
```suggestion
### Remove a cluster
```

File: calico-cloud/operations/cluster-management.mdx [hunk 6]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
+
+1. In Manager UI, go to **Managed Clusters**.
+1. For the cluster you want to delete, select **Actions** > **Delete**.
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx]:
```suggestion
1. For the cluster you want to remove, select **Actions** > **Delete**, and then confirm your choice by selecting **Delete cluster**.
```

File: calico-cloud/operations/cluster-management.mdx [hunk 7]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
+1. Verify details on the confirmation modal and click on **Delete** to confirm the managed cluster deletion.
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx]: rm

File: calico-cloud/operations/cluster-management.mdx [hunk 8]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
+
+![delete managed cluster](/img/calico-cloud/managed-clusters-delete.png)
+![delete managed cluster modal](/img/calico-cloud/managed-clusters-delete-step1.png)
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx]: rm 

I don't think images add anything here.

File: calico-cloud/operations/cluster-management.mdx [hunk 9]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
+
+:::note
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx:22]: This note raises questions.

How do I remove logs?

File: calico-cloud/operations/cluster-management.mdx [hunk 10]
```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
...
+
+Deleting a managed cluster does not remove {{prodname}} logs.
+
+If a new managed cluster is connected with the same name as a previously deleted managed cluster, the logs associated with the deleted managed cluster will be displayed as if they belong to the new managed cluster.
+
+:::
+
+### Recover a deleted managed cluster
```
Comment (ctauchen) [inline calico-cloud/operations/cluster-management.mdx:30]: I don't understand how this works. If we've removed the cluster from the list, how do we find it again?

**Review by ctauchen — COMMENTED** [review]
Thanks for this -- a few notes and a question or two @jaderhs

File: calico-cloud/operations/cluster-management.mdx (hunk 9 above)
Comment (jaderhs) [reply calico-cloud/operations/cluster-management.mdx:22]: It is not possible to remove logs at the moment. The user will be able to view these logs in Kibana

File: calico-cloud/operations/cluster-management.mdx (hunk 2 above)
Comment (jaderhs) [reply calico-cloud/operations/cluster-management.mdx]: Sounds good, updated https://github.com/tigera/docs/pull/961/commits/ccd5b72a48cbbaa8a3a18f47c1787fee68a5779e

File: calico-cloud/operations/cluster-management.mdx (hunk 3 above)
Comment (jaderhs) [reply calico-cloud/operations/cluster-management.mdx]: Item updated https://github.com/tigera/docs/pull/961/commits/f11f949cc5a80c156000fba8ec494143384c5a10

File: calico-cloud/operations/cluster-management.mdx (hunk 4 above)
Comment (jaderhs) [reply calico-cloud/operations/cluster-management.mdx]: Item updated https://github.com/tigera/docs/pull/961/commits/3ae147cdd1e8883739cc8aec4233aa4e82340c0f

File: calico-cloud/operations/cluster-management.mdx (hunk 7 above)
Comment (jaderhs) [reply calico-cloud/operations/cluster-management.mdx]: Numbered list removed https://github.com/tigera/docs/pull/961/commits/099d05246d1e96222efb851cf0ce7fed0c32f97a

File: calico-cloud/operations/cluster-management.mdx (hunk 8 above)
Comment (jaderhs) [reply calico-cloud/operations/cluster-management.mdx]: Images removed https://github.com/tigera/docs/pull/961/commits/b0c5e3fbb4d4b272c2931824f5210c34c0f5aa0e

File: calico-cloud/operations/cluster-management.mdx (hunk 10 above)
Comment (jaderhs) [reply calico-cloud/operations/cluster-management.mdx:30]: The procedure is incorrect, as you mentioned the cluster will be removed from the list.
The correct procedure is to connect the cluster (with the same name as the previously deleted) https://github.com/tigera/docs/pull/961/commits/877124c80c729c5ae25d3a24696e88e08a885ea6

**Pull request merged**